src/stdlib_math.fypp

```diff
@@ -0,0 +1,35 @@
+#:include "common.fypp"
+
+#:set INT_KINDS_TYPES = [("int8", "integer"), ("int16", "integer"), ("int32", "integer"), ("int64", "integer")]
+#:set REAL_KINDS_TYPES = [("sp", "real"), ("dp", "real"), ("qp", "real")]
+#:set IR_KINDS_TYPES = INT_KINDS_TYPES + REAL_KINDS_TYPES
```

Comment:
Why did you re-define these variables? why not use the ones provided in common.fypp?